index.js

```diff
@@ -127,25 +144,27 @@ const drawCommand = (message) => {
   });
 };
 
-/*
-const inventoryCommand = () => {
-
-if user draws a card
-store name of card to firebase under the key cardName
-when the same user draws a second card, store the name of the second card under the previous draw within cardName
-continue to do this to all additional cards
-
-user{
-  userID{
-    lastDrawTime: xxxx
-    username: ""
-    cardName: "Blue Eyes White Dragon", "Red Eyes Black Dragon"
-  }
-}
-
+const inventoryCommand = (message) => {
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    let userInventory = message.author.id;
+    let inventoryRef = "inventory";
+    let inventoryNames;
+    let userInventoryAccess = userData[userInventory][inventoryRef];
+    let cardID = Object.keys(userInventoryAccess);
+    let finalStringInventory = "";
+
+    for (let i = 0; i < cardID.length; i++) {
+      let keyCard = cardID[i];
+
+      inventoryNames = "•  " + userInventoryAccess[keyCard]["name"];
```

Comment:
you can simplify by adding the right side directly to `finalStringInventory`, which would enable you to get rid of `inventoryNames`